ddtrace/tracer/option.go

```diff
@@ -288,6 +288,12 @@ func SpanType(name string) StartSpanOption {
 	return Tag(ext.SpanType, name)
 }
 
+// MeasureSpan adds the measured key to a span, thereby making it eligible for
```

Comment:
The "measured key" doesn't mean much to the user. It's an implementation detail. We should describe what this does from a functional standpoint:
```suggestion
// MeasureSpan marks this span to be measured for metrics and stats calculations.
```